Cargo.toml

```diff
@@ -39,23 +39,24 @@ pam-auth = { package = "pam", version = "0.7", optional = true }
 hyper = { version = "0.13", optional = true }
 url = { version = "1.7.2", optional = true }
 http = { version = "0.2.1", optional = true }
-serde = { version = "1.0.98", optional = true, features = ["derive"] }
-serde_json = { version = "1.0.40", optional = true }
+serde = { version = "1.0.106", optional = true, features = ["derive"] }
+serde_json = { version = "1.0.51", optional = true }
 path_abs = "0.4"
 prometheus = "0.7"
 uuid = { version = "0.7", features = ["v4"] }
 rand = "0.6"
 
 hyper-rustls = {version = "0.19.1", optional = true}
 yup-oauth2 = {version = "4.1.0", optional = true}
-mime = {version = "0.3.13", optional = true}
+mime = {version = "0.3.16", optional = true}
 itertools = "0.9"
 
 [dev-dependencies]
 tempfile = "3"
 ftp = "3"
 pretty_env_logger = "0.2"
 pretty_assertions = "0.5"
+tokio = { version = "0.2.17", features = ["rt-threaded"]}
```

Comment:
Didn't know you can specify it twice. Cool.

Reply:
I didn't know it either, but I had to, and it worked :)